### DISCUSSION TITLE
Home Page Layout

### DISCUSSION THREAD

**Discussion by Suvink** [gcsrilanka/peliculopedia#17] [Ideas]
In the current design, the content loads in the middle section of a wrapper which covers 3 sides of the main window. Since this navigation is applied to all the pages by default, some pages like login looks a bit off and hard to manage the responsiveness. 
![Screenshot 2021-10-19 at 21 20 34](https://user-images.githubusercontent.com/10671497/137946654-3ebcc49d-eb19-4190-a5d6-cb6ccd7ffefc.png)
 
So I was thinking, maybe we can separate those 3 components into Navbar, Left bar and Right bar and include in the necessary pages separately. It can give is the flexibility to add different kinds of layouts to the screens. 

What do you guys think? 
@AhamedR @isuruAb @Safnaj @imakish

**Comment by AhamedR** [comment]
Nice catch @Suvink, I think in this case we can handle this scenario in 2 ways.
1.  We can design separate components for each side as @Suvink mentioned.
2. We can manage this through the `Route switch`, e.g. 

![image](https://user-images.githubusercontent.com/24555001/137956166-f71df13b-b78a-4531-baae-453a24d8d144.png)
 as mentioned in this image we can wrap only the relevant routes. 
 
 The cons of using the second option.
 - we will only have the Login page. event the navbar will be ignored.

**Comment by Suvink** [comment]
Yeah thats true and I saw you already added that in #18. So @AhamedR  my only concern was if we decide to add more types of layouts in the future, this might cause problems but I went through your code and seems like this options is good.

**Reply by AhamedR** [reply]
Thanks, @Suvink and If you think this might cause any issue we can have a better structuring. Any suggestions?
